app/src/main/java/com/framgia/pokemon/data/source/PokemonDataSource.java

```diff
@@ -0,0 +1,17 @@
+package com.framgia.pokemon.data.source;
+
+import com.framgia.pokemon.data.model.Pokemon;
+import com.framgia.pokemon.data.model.response.PokemonResponse;
+
+import io.reactivex.Observable;
+import retrofit2.Call;
+
+/**
+ * Created by ducpm on 25/08/17.
+ */
+
+public interface PokemonDataSource {
+   Observable<Call<PokemonResponse>> obtainListPokemon();
```

Comment:
ERROR: 'method def modifier' have incorrect indentation level 3, expected level should be 4.